src/modules/TodosContainer.js

```diff
@@ -0,0 +1,26 @@
+import Todo from './Todo.js';
+import { getTodos, setTodos } from './store.js';
+import UI from './UI.js';
+
+export default class TodosContainer {
+  todos = getTodos() || [];
+
```

Comment:
- [ ] As reqested by the previous reviewer and the [project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20editing%20task%20descriptions.), please implement the editing function. 🙏 

_Thank you for communicating with the reviewers ❤️ However, please keep the project requirement as the first resource to get back to when you're confused between the required and the optional tasks._

Reply:
@MahmoudBakr23 I will do that, thanks for the review 🙏 .